<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoardEmulatorWidget</class>
 <widget class="QWidget" name="BoardEmulatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>value</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QTextEdit" name="inputValueTextEdit"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>offset</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>send</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QTextEdit" name="outputValueTextEdit"/>
   </item>
   <item row="1" column="0" rowspan="2" colspan="2">
    <widget class="QTextEdit" name="offsetTextEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
